client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -100,9 +98,21 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 									required={ fieldName !== 'state' }
 									value={ this.state[ fieldName ] }
 								/>
+								{ fieldName !== 'state' && (
+									<FormInputValidation
+										isError={ ! this.state[ fieldName ] }
+										text={ translate( 'Required field.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Required Fields' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).